includes/Vendor/SetupWizard.php

```diff
@@ -118,6 +118,9 @@ public function frontend_enqueue_scripts() {
         wp_enqueue_script( 'jquery-tiptip' );
         wp_enqueue_script( 'jquery-blockui' );
         wp_enqueue_script( 'wc-enhanced-select' );
+
+        // Load map scripts.
+        dokan()->scripts->load_gmap_script();
     }
```

Comment:
Please check if  map box loads properly.

Reply:
> Please check if map box loads properly.

Bhai, both `Map Box` and `Google Map` are loading properly.